packages/fxa-settings/src/components/Settings/FlowSetupRecoveryPhoneSubmitNumber/index.tsx

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import FlowContainer from '../FlowContainer';
+import ProgressBar from '../ProgressBar';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import Banner from '../../Banner';
+import { BackupRecoveryPhoneImage } from '../../images';
+import { getLocalizedErrorMessage } from '../../../lib/error-utils';
+import { useFtlMsgResolver } from '../../../models';
+import FormPhoneNumber from '../../FormPhoneNumber';
+
+export type FlowSetupRecoveryPhoneSubmitNumberProps = {
+  localizedBackButtonTitle: string;
+  localizedPageTitle: string;
+  navigateBackward: () => void;
+  navigateForward: () => void;
+  verifyPhoneNumber: (phoneNumber: string) => Promise<void>;
+};
+
+export const FlowSetupRecoveryPhoneSubmitNumber = ({
+  localizedBackButtonTitle,
+  localizedPageTitle,
+  navigateBackward,
+  navigateForward,
+  verifyPhoneNumber,
+}: FlowSetupRecoveryPhoneSubmitNumberProps) => {
+  const [localizedErrorBannerMessage, setLocalizedErrorBannerMessage] =
+    useState('');
+
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const clearBanners = async () => {
+    return new Promise<void>((resolve) => {
+      setLocalizedErrorBannerMessage(''); // Clear the banner message
+      setTimeout(() => resolve(), 0); // Wait for DOM update
+    });
+  };
+
+  const handlePhoneNumber = async (phoneNumber: string) => {
+    await clearBanners();
+    try {
+      await verifyPhoneNumber(phoneNumber);
+      navigateForward();
+      return { hasErrors: false };
+    } catch (error) {
+      const localizedError = getLocalizedErrorMessage(ftlMsgResolver, error);
+      setLocalizedErrorBannerMessage(localizedError);
+      return { hasErrors: true };
+    }
+  };
+
+  return (
+    <FlowContainer
+      {...{ localizedBackButtonTitle }}
+      title={localizedPageTitle}
+      onBackButtonClick={navigateBackward}
+    >
+      <ProgressBar currentStep={1} numberOfSteps={2} />
+      {localizedErrorBannerMessage && (
+        <Banner
+          ref={(el) => el && el.focus()}
```

Comment:
Do we need to focus on the banner? When I use a screen reader I then have to tab through again to reach the phone input. It's immediately read, which is what sighted users will do when they see it pop up.

Reply:
I'd opted to focus on the banner to ensure the banner is in sight for sighted users, but perhaps focusing on the input it better? Tricky with this design since the two elements might not both be in view on smaller devices. I'll switch to focusing on the input and will send a link for UX review (post-holiday)